debian/control

```diff
@@ -624,3 +626,142 @@ Description: Enhanced system logging daemon (example plugins)
     AMQP), files or databases (like PostgreSQL or MongoDB).
  .
  This package provides a collection of example plugins.
+
+Package: syslog-ng-mod-java
```

Comment:
no, never. I don't think this should be downstreamed, we can do keep it in upstream, so user-built compilations will have java support.

it would be great to package these in debian, but I can imagine the nightmare it would become. I am not even sure our java dependencies described in gradle even exist as debian packages.